USERS.md

```diff
@@ -44,3 +44,4 @@ Organizations below are **officially** using Argo Rollouts. Please send a PR wit
 1. [Ubie](https://ubie.life/)
 1. [VISITS Technologies](https://visits.world/en)
 1. [Yotpo](https://www.yotpo.com/)
+1. [DaoCloud](https://daocloud.io)
```

Comment:
Can this be put in alphabetical order.

Reply:
Ended up changing this after looking that the failing tests

Reply:
Sorry for note this late, as this PR merged, I will update it in another one later.

Reply:
Sorry again, I saw you have done it, thanks so much.